HGustavs/LenaSYS
# Fixed the colour selector functionality #11983

- #11983
# Fixes :
- Fixed the title from BG Color to Color
- Fixed so that the color selector div is aligned with the background colour option.
- Fixed so that the color selector doesn't remain open when options pane is closed.

## To test: 
### **1. Make sure the title has changed from _BG Color_ to _Color_**
<img width="348" alt="colorTitle" src="https://user-images.githubusercontent.com/81613484/166419886-85339ff4-7c63-43f2-a27b-23454ccd4fa5.png">

### **2. Make sure the color selector is properly aligned beside the _"Color"_ option.** and not the _"Stroke color"_
<img width="600" alt="alignment" src="https://user-images.githubusercontent.com/81613484/166419420-0314b1a5-397f-4c4a-a212-4221a7a33049.png">
 
### **3. Make sure the color selector doesn't remain open when closing the option pane**
**Test by:**
- Press an entity in the diagram
- Open up the option pane
- Click on the color option
- Close the option pane by pressing the title named "Options" while the color selector is still open
- Make sure the color selector is also closed 

It should **NOT** look like this:
<img width="138" alt="NOTlikethis" src="https://user-images.githubusercontent.com/81613484/166420513-0fcf2824-7a6a-4e2a-b410-85dec7b5e14e.png">
